Library/Homebrew/utils/github.rb

```diff
@@ -487,7 +488,10 @@ def create_bump_pr(info, args:)
       changed_files += additional_files if additional_files.present?
 
       if args.dry_run? || (args.write? && !args.commit?)
-        ohai "try to fork repository with GitHub API" unless args.no_fork?
+        unless args.no_fork?
+          ohai "try to fork repository with GitHub API"
+          ohai "..into `#{args.fork_org}` GitHub organization" if args.fork_org
```

Comment:
```suggestion
          puts "into `#{args.fork_org}` GitHub organization" if args.fork_org
```

or something as the `ohai ".."` is a bit weird.